qds_sdk/commands.py

```diff
@@ -136,6 +150,19 @@ def get_log(self):
         r = conn.get_raw(log_path)
         return r.text
 
+    def get_log_partial(self, err_pointer=0, tmp_pointer=0):
+        """
```

Comment:
nop, when someone imports sdk, they access this method directly. we can add extra args without breaking them, but response is changing as well here, which might break existing tools.